src/Files.App/Views/LayoutModes/GridViewBrowser.xaml.cs

```diff
@@ -191,17 +192,30 @@ override public void StartRenameItem()
 			}
 			else
 			{
-				TextBlock textBlock = gridViewItem.FindDescendant("ItemName") as TextBlock;
+				TextBlock textBlockName = gridViewItem.FindDescendant("ItemName") as TextBlock;
```

Comment:
Aren't the xaml changes enough?

Reply:
What do you mean? Renaming this variable?

Reply:
I mean, can't this be done via xaml by setting the RowSpan on the TextBox?

Reply:
First I set that, but the other elements were visible in front of it, and beside the edges.

Reply:
You might have to change the `Canvas.ZIndex` property on the TextBox.

Reply:
I reverted the name changes and deleted the unnecessary visibility changes.